hypha/apply/funds/models/submissions.py

```diff
@@ -789,10 +799,30 @@ def prepare_search_values(self):
                 else:
                     yield value
 
+    def prepare_search_values(self):
+        self.get_searchable_contents()
+
         # Add named fields into the search index
         for field in ['full_name', 'email', 'title']:
             yield getattr(self, field)
 
+    def index_components(self):
+        return {
+            'A': self.title,
+            'C': " ".join([self.full_name, self.email]),
+            'B': ' '.join(self.get_searchable_contents()),
```

Comment:
Maybe we can add an entry here with `f'#{self.id}'`?

Reply:
See: https://github.com/HyphaApp/hypha/pull/3149/commits/5148324a3ea1d1aa6bcdf8003e913de6a7f5e53b

I would prefer it to be `id:<>` instead of `#<num>`, similar to the syntax used by gmail and Github. This syntax is expandable too.